ynr/apps/api/tests/test_api_v09.py

```diff
@@ -292,9 +293,11 @@ def test_api_version_info(self):
         self.assertEqual(info["users_who_have_edited"], 0)
         self.assertEqual(info["interesting_user_actions"], 0)
 
-        LoggedAction.objects.create(action_type="set-candidate-not-elected")
+        LoggedAction.objects.create(
+            action_type=ActionType.SET_CANDIDATE_NOT_ELECTED
+        )
 
-        LoggedAction.objects.create(action_type="edit-candidate")
```

Comment:
@symroe This failed until I changed it to an existing choice. Is this the test you're referring to?